pkg/metrics/metrics.go

```diff
@@ -177,9 +179,10 @@ var (
 )
 
 type subscriptionSyncLabelValues struct {
-	installedCSV string
-	pkg          string
-	channel      string
+	installedCSV     string
+	pkg              string
+	channel          string
+	approvalStrategy string
```

Comment:
Since we only support two approval methods (manual and automatic) could we use a boolean here to minimize the data we send and store?

Reply:
We could, i had this in mind, but so far all prometheus labels have string values.  Even booleans are stored as "true" or "false" string.

Reply:
we can store this as "automaticApproval": "!" if automatic, "0" if it set to manual

Reply:
This is fine.